### PR TITLE
Add support for forc 0.13

### PR DESCRIPTION
**Pull request by digorithm**

### CODE REVIEW

**Review by vnepveu — APPROVED** [review]
:shipit:

File: packages/fuels-abigen-macro/tests/test_projects/two_structs/src/main.sw [hunk 1]
```diff
@@ -9,8 +9,8 @@ pub struct StructTwo {
 }
 
 abi MyTest {
-    fn something(input: StructTwo) -> u64;
-    fn something_else(input: StructOne) -> u64;
+    fn something(input: StructOne) -> u64;
+    fn something_else(input: StructTwo) -> u64;
```
Comment (adlerjohn) [inline packages/fuels-abigen-macro/tests/test_projects/two_structs/src/main.sw:13]: Why were these changed?

Comment (digorithm) [reply packages/fuels-abigen-macro/tests/test_projects/two_structs/src/main.sw:13]: Because before it was wrong but somehow the compiler never complained 😂 They have the same inner structure (just a `u64`) so I think the compiler was okay using them interchangeably? This was still weird but we never caught it. But now the compiler actually complains, hence why I changed it.

**Review by MujkicA — DISMISSED** [review]

**Review by adlerjohn — COMMENTED** [review]
CI failing

**Review by vnepveu — APPROVED** [review]
I was kind of sure the error message was the one I had changed haha
